package.json

```diff
@@ -15,6 +16,6 @@
     "mocha-junit-reporter": "1.6.1"
   },
   "scripts": {
-    "test": "grunt verify --force"
+    "test": "./node_modules/mocha/bin/mocha test/generic.spec.js"
```

Comment:
That is not correct way of running tests and we are not allowing to use it that way in Jenkins.